MPChartLib/src/main/java/com/github/mikephil/charting/renderer/RadarChartRenderer.java

```diff
@@ -275,21 +286,44 @@ protected void drawWeb(Canvas c) {
 
         int labelCount = mChart.getYAxis().mEntryCount;
 
-        MPPointF p1out = MPPointF.getInstance(0,0);
-        MPPointF p2out = MPPointF.getInstance(0,0);
-        for (int j = 0; j < labelCount; j++) {
+        MPPointF p1out = MPPointF.getInstance(0, 0);
+        MPPointF p2out = MPPointF.getInstance(0, 0);
 
+        for (int j = 0; j < labelCount; j++) {
+            if (mChart.isCustomLayerColorEnable()) {
+                innerArea.rewind();
+                paint.setColor(mChart.getLayerColorList().get(j));
+            }
             for (int i = 0; i < mChart.getData().getEntryCount(); i++) {
-
                 float r = (mChart.getYAxis().mEntries[j] - mChart.getYChartMin()) * factor;
 
                 Utils.getPosition(center, r, sliceangle * i + rotationangle, p1out);
                 Utils.getPosition(center, r, sliceangle * (i + 1) + rotationangle, p2out);
 
                 c.drawLine(p1out.x, p1out.y, p2out.x, p2out.y, mWebPaint);
+                if (mChart.isCustomLayerColorEnable()) {
+                    if (p1out.x != p2out.x) {
+                        if (i == 0) {
+                            innerArea.moveTo(p1out.x, p1out.y);
+                        } else {
+                            innerArea.lineTo(p1out.x, p1out.y);
+                        }
+                        innerArea.lineTo(p2out.x, p2out.y);
+                    }
+                }
 
 
             }
+            if (mChart.isCustomLayerColorEnable()) {
+                temp.set(innerArea);
+                if (!innerArea.isEmpty()) {
+                    boolean result = innerArea.op(previousPath, Path.Op.DIFFERENCE);
```

Comment:
This comes with this lint error
![image](https://user-images.githubusercontent.com/3314607/157013484-2d8a4a70-9529-474e-bdc3-0b7652c1b608.png)

<img width="1060" alt="image" src="https://user-images.githubusercontent.com/3314607/157013711-28de0cb8-d5ac-4de4-a5d8-5d3765584aad.png">


I solved and merged it in my updated and maintained [fork](https://github.com/AppDevNext/MPAndroidChart?organization=AppDevNext&organization=AppDevNext
)